pkgs/servers/sql/postgresql/libpq.nix

```diff
@@ -0,0 +1,70 @@
+{ stdenv
+, lib
+, openssl
+, zlib
+, libkrb5
+, icu
+, postgresql
+, pkg-config
+, tzdata
+, gssSupport ? with stdenv.hostPlatform; !isWindows && !isStatic
+}:
+
+stdenv.mkDerivation {
+  pname = "libpq";
+  version = postgresql.version;
+  src = postgresql.src;
+
+  configureFlags = [
+    "--with-openssl"
+    "--with-icu"
+    "--without-readline"
+    "--with-system-tzdata=${tzdata}/share/zoneinfo"
+  ]
+  ++ lib.optionals gssSupport [ "--with-gssapi" ];
+
+  nativeBuildInputs = [ pkg-config tzdata ];
+  buildInputs = [ openssl zlib icu ]
+    ++ lib.optional gssSupport libkrb5;
+
+  buildPhase = ''
+    runHook preBuild
+
+    make -C src/bin/pg_config
```

Comment:
There is no other way to build this?
It looks a strange thing to me: 

You are basically doing a `foreach`-like loop.
What happens when you try to run each of them separately?
Can you install each of them separately?

Reply:
Turns out there's a `submake-libpq` makefile build target. But there's no equivalent for installing these unfortunately.